runtime/compiler/CMakeLists.txt

```diff
@@ -95,7 +90,6 @@ if(J9VM_OPT_JITSERVER)
 		set(OPENSSL_ROOT_DIR ${OPENSSL_DIR})
 	endif()
 
-	include(FindOpenSSL)
 	find_package(OpenSSL REQUIRED)
```

Comment:
I take it `include(FindOpenSSL)` is just redundant?

Reply:
yes. `find_package(x)` automatically includes `FindX`